File: c/darts/darts.c
#include "darts.h"
#include <math.h>

uint8_t score(coordinate_t landing_position) {
    float d = sqrt(pow(landing_position.x, 2) + pow(landing_position.y, 2));
    if (d <= 1.0f) return 10;
    if (d <= 5.0f) return 5;
    if (d <= 10.0f) return 1;
    
    return 0;
}

File: python/darts/darts.py
def score(x, y):
    distance = (x**2 + y**2) ** 0.5 
    if distance <= 1.0:
        return 10
    elif distance <= 5.0:
        return 5
    elif distance <= 10.0:
        return 1
    else:
        return 0
